actix-web/src/types/path.rs

```diff
@@ -53,6 +53,26 @@ use crate::{
 ///     format!("Welcome {}!", info.name)
 /// }
 /// ```
+///
+/// Segments matching multiple path components can be deserialized
+/// into a Vec<_> to percent-decode the components individually. Empty
+/// path components are ignored.
+///
+/// ```
+/// use actix_web::{get, web};
+/// use serde::Deserialize;
+///
+/// #[derive(Deserialize)]
+/// struct Tail {
+///     tail: Vec<String>,
+/// }
+///
+/// // extract `Tail` from a path using serde
+/// #[get("/path/to/{tail:.*}")]
```

Comment:
please can we use the suggested syntax for tail matches in the tests and docs
https://docs.rs/actix-router/latest/actix_router/struct.ResourceDef.html#tail-segments

```suggestion
/// #[get("/path/to/{tail}*")]
```